pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3148,6 +3148,12 @@ type DisruptionManagementSpec struct {
 	// +optional
 	OSDMaintenanceTimeout time.Duration `json:"osdMaintenanceTimeout,omitempty"`
 
+	// The maximum number of OSDs to update in parallel.
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:default=20
+	// +optional
+	OSDMaxUpdatesInParallel uint32 `json:"osdMaxUpdatesInParallel,omitempty"`
```

Comment:
I can see the thought process to put it here since upgrading OSDs can be disrupting, but the disruption management is intended for settings related to PDB management. So I would vote to put this in the `StorageScopeSpec`, aka `storage` in the yaml, with other OSD-specific settings.